benchmark/worker/benchmark_client.go

```diff
@@ -260,36 +265,44 @@ func (bc *benchmarkClient) doCloseLoopUnary(conns []*grpc.ClientConn, rpcCountPe
 				// Now relying on worker client to reserve time to do warm up.
 				// The worker client needs to wait for some time after client is created,
 				// before starting benchmark.
-				done := make(chan bool)
-				for {
-					go func() {
-						start := time.Now()
-						if err := benchmark.DoUnaryCall(client, reqSize, respSize); err != nil {
+				if poissonLambda == nil { // Closed loop.
```

Comment:
Maybe add a small comment in here to explain what the closed loop means? Also one below for the closed loop?